movies/ui_movies.py

```diff
@@ -1,5 +1,51 @@
+import time
+import ui.main as ui
+import csv
+
 def call_add_movie():
-    pass
+    ui.clear()
+    print('Add Movie')
+    print()
+    movie_title = input("Movie Title: ")
+    movie_release_date = input("Movie Release Date: ")
+    rating = input("Rating: ")
```

Comment:
@simpletech310 Note completely sure, but I think only integer values should be accepted. What are your thoughts?